server/database.js

```diff
@@ -58,6 +58,8 @@ class Database {
         "patch-monitor-expiry-notification.sql": true,
         "patch-status-page-footer-css.sql": true,
         "patch-added-mqtt-monitor.sql": true,
+        "patch-monitor-add-resend-interval.sql": true,
+        "patch-heartbeat-add-last-notified-time.sql": true,
```

Comment:
Can you merge this into one SQL file?

Reply:
Of course, I thought each table should have its own patch file.

Thanks for making it clear how it is desired 👍